src/yacht/hypothesis_recovery_src.py

```diff
@@ -24,17 +25,17 @@
     sys.stdout, format="{time:YYYY-MM-DD HH:mm:ss} - {level} - {message}", level="INFO"
 )
 
+# Set up contants
 SIG_SUFFIX = ".sig"
-
+FILE_LOCATION = os.path.dirname(os.path.realpath(__file__))
 
 def get_organisms_with_nonzero_overlap(
     manifest: pd.DataFrame,
     sample_file: str,
-    scale: int,
-    ksize: int,
     num_threads: int,
     path_to_genome_temp_dir: str,
     path_to_sample_temp_dir: str,
+    num_genome_threshold: int = 1000000
```

Comment:
This argument doesn't seem to be documented. Is the intent of this just to calculate the number of passes @mahmudhera ? Or sets some other threshold with respect to number of organisms detected or the like?

Reply:
@dkoslicki, this parameter is not expected to be modified by the users. Or probably I can always set it to 1000000. It is not associated with the YACHT results. It is just associated with how many genomes are processed in a block within Mahmudera's algorithm. So, don't worry about it.

Reply:
Yes, understood. But 2 years from now when we want to make yet further changes, it should probably be documented in the code with a comment about what it's doing

Reply:
I have added documents for these new parameters. Please check the new commit.

Reply:
I don't quite remember doing this, I guess @chunyuma has already addressed this